src/sentry/snuba/errors.py

```diff
@@ -179,3 +188,165 @@ def timeseries_query(
         params["end"],
         rollup,
     )
+
+
+def top_events_timeseries(
+    timeseries_columns: list[str],
+    selected_columns: list[str],
+    user_query: str,
+    params: ParamsType,
+    orderby: list[str],
+    rollup: int,
+    limit: int,
```

Comment:
I'm moving to add snuba_params everywhere, do you mind adding it here too?
reference here; https://github.com/getsentry/sentry/compare/wmak/ref/change-snuba-modules-to-include-snuba-params?expand=1